### PR TITLE
TR4i

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

**Comment by ricardopereira** [comment]
RTL2f is currently pending because Ably service is missing this feature but I will inject a message with `flags` with `RESUMED`.

File: Spec/RealtimeClientChannel.swift [hunk 1]
```diff
@@ -356,6 +356,40 @@ class RealtimeClientChannel: QuickSpec {
...
+                    attachedMessage.action = .Attached
+                    attachedMessage.channel = "test"
+                    attachedMessage.flags = 4 //Resumed
+
+                    waitUntil(timeout: testTimeout) { done in
+                        channel.once(.Update) { stateChange in
+                            guard let stateChange = stateChange else {
+                                fail("ChannelStageChange is nil"); done(); return
+                            }
+                            expect(stateChange.resumed).to(beTrue())
```
Comment (mattheworiordan) [inline Spec/RealtimeClientChannel.swift]: Mind doing one test the other way around please?

Comment (mattheworiordan) [inline Spec/RealtimeClientChannel.swift]: In fact, might be worth checking that the resumed flag is always false on the first attach

Comment (ricardopereira) [inline Spec/RealtimeClientChannel.swift]: Done e70fb41.

**Comment by ricardopereira** [comment]
@mattheworiordan PTAL

**Pull request merged**